pages/index.js

```diff
@@ -23,42 +23,114 @@ export const getStaticProps = async () => {
             url
           }
         }
-        posts(sort: "created_at:desc", limit: 5) {
```

Comment:
@VamshiGangadhar if there is limit, replace it with pagination: {page: 0, limit: 5}